CHANGELOG-LATEST.md

```diff
@@ -1,33 +1,36 @@
-### New Features
+### New features
 *   📱 Initial support for cross-platform RevenueCat Paywalls 🐾 🧱  (#852) 
 
 #### Instructions:
-- For Android, you need to change your `MainActivity` to subclass `FlutterFragmentActivity` instead of `FlutterActivity`.
-- Update `purchases-flutter` in your `pubspec.yaml`:
+- For Android, you need to change your `MainActivity` to subclass `FlutterFragmentActivity` instead of `FlutterActivity`. Also, the min sdk version of the new package is 24. Please make sure your app's android/build.gradle minSdkVersion has that or a higher version.
+- Add `purchases-ui-flutter` in your `pubspec.yaml`:
 ```yaml
 dependencies:
-  purchases_flutter: 6.6.0-beta.3
+  purchases_ui_flutter: 6.15.0-beta.4
 ```
 
 #### Usage:
 ```dart
-await Purchases.presentPaywallIfNeeded("pro");
+import 'package:purchases_ui_flutter/purchases_ui_flutter.dart';
+
+await RevenueCatUI.presentPaywallIfNeeded("pro");
 ```
 
 #### Limitations:
-
 - Currently only full screen paywalls are supported
 - There is no way to detect paywall events other than using `addCustomerInfoUpdateListener`
-- Android's `minSdkVersion` is temporarily increased from `19` to `24` to support paywalls. This will be reverted in a future release as we split `purchases_flutter` and `purchases_ui_flutter`
-
-### Breaking changes from previous beta
 
-- `PurchasesFlutterActivity` has been removed. Use `FlutterFragmentActivity` provided by Flutter instead
-
-### Fixes from previous beta
-* Fix `presentPaywallIfNeeded` (#904) via Toni Rico (@tonidero)
+#### Breaking changes from previous beta:
+- Now, you need to add a new dependency `purchases_ui_flutter` to your `pubspec.yaml`. Before this, it was part of the main RevenueCat SDK `purchases_flutter`.
```

Comment:
I would rephrase this with something like: Paywalls functionality has been extracted into a new dependency called purchases_ui_flutter. Add purchases_ui_flutter to your pubspec.yaml. Before this release, it was part...

Or something like that to make it more clear